wizard/templates/wizard/data/editor.html

```diff
@@ -39,8 +39,8 @@
             </p>
 
             <p>
-                Only <code>automl, csv and arff</code> archives are currently supported. They are defined as a zip
-                file containing at least:
+                Only <code>AutoML, CSV and ARFF</code> archives are currently supported. All archives will be converted
```

Comment:
Should probably be:

`<code>AutoML</code>, <code>CSV</code> and <code>ARFF</code>`